src/Microsoft.ML.Experimental/MLContextExtensions.cs

```diff
@@ -11,5 +14,105 @@ public static class MLContextExtensions
         /// <param name="ctx"><see cref="MLContext"/> reference.</param>
         public static void CancelExecution(this MLContext ctx) => ctx.CancelExecution();
+
+        /// <summary>
+        /// Normalize (rescale) the column according the <see cref="NormalizingEstimator.NormalizationMode.MinMax"/> mode.
+        /// It normalizes the data based on the observed minimum and maximum values of the data.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="ensureZeroUntouched">Whether to map zero to zero, preserving sparsity.</param>
+        /// <param name="maximumExampleCount">Maximum number of examples used to train the normalizer.</param>
+        public static NormalizingEstimator NormalizeMinMax(this TransformsCatalog catalog,
+           string outputColumnName, string inputColumnName = null,
+           bool ensureZeroUntouched = NormalizingEstimator.Defaults.EnsureZeroUntouched,
+           long maximumExampleCount = NormalizingEstimator.Defaults.MaximumExampleCount)
+        {
+            var columnOptions = new NormalizingEstimator.MinMaxColumnOptions(outputColumnName, inputColumnName, maximumExampleCount, ensureZeroUntouched);
+            return new NormalizingEstimator(CatalogUtils.GetEnvironment(catalog), columnOptions);
+        }
+
+        /// <summary>
+        /// Normalize (rescale) the column according the <see cref="NormalizingEstimator.NormalizationMode.MeanVariance"/> mode.
+        /// It normalizes the data based on the computed mean and variance of the data.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="ensureZeroUntouched">Whether to map zero to zero, preserving sparsity.</param>
+        /// <param name="maximumExampleCount">Maximum number of examples used to train the normalizer.</param>
+        /// <param name="useCdf">Whether to use CDF as the output.</param>
+        public static NormalizingEstimator NormalizeMeanVariance(this TransformsCatalog catalog,
+            string outputColumnName, string inputColumnName = null,
+            bool ensureZeroUntouched = NormalizingEstimator.Defaults.EnsureZeroUntouched,
+            long maximumExampleCount = NormalizingEstimator.Defaults.MaximumExampleCount,
```

Comment:
I noticed that you reversed the order of `maximumExampleCount` and `ensureZeroUntouched` here and in the other extensions compared to the order in which they appeared in the constructors of the `ColumnOptions`,
It seems that `maximumExampleCount` is more important than the other argument, or at least  I would expect people to use it more.

Reply:
good point. 

Subject to  whether the internal constructors use them, lets keep the following order : 

- maximumExampleCount  
- fixZero 
- useCdf
- maximumBinCount
- mininimumExamplesPerBin

---
In reply to: [268249207](https://github.com/dotnet/machinelearning/pull/3052#discussion_r268249207) [](ancestors = 268249207)